release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -114,6 +112,29 @@ self-signed certificates for authorization.
 
 //For more information, see Installing a cluster on OpenStack using your own infrastructure (link when published).
 
+[id="ocp-4-4-openstack-validates-rhcos-images-checksum"]
+==== OpenStack validates {op-system} images by checking sha256 checksum
+
+On {rh-openstack}, the installer now performs automatic checksum validation of
+{op-system} images.
+
+[id="ocp-4-4-support-for-east-west-traffic-with-ovn-on-openstack-kuryr"]
+==== Support for east-west traffic with OVN load balancing on OpenStack with Kuryr
+
+{product-title} installations that use Kuryr on {rh-openstack} 16 can now use
+the OVN load-balancing provider driver instead of the Amphora driver. If OVN and
+the OVN Octavia driver are present in the environment, OVN is used
+automatically. As a result, load balancer performance and resource utilization
+are improved. The need for a load balancer VM for each service is also removed.
+
+[id="ocp-4-4-using-upgrade-channels"]
+==== Using upgrade channels for 4.4.0 release
+
+Upgrades from {product-title} 4.3.15 and later to 4.4.0 will be available in the
```

Comment:
Per our slack convo earlier, just double check on 4.3.15 or whether it'll be 4.3.16.

Reply:
Good call! I'll double check.

Reply:
Verified :+1: